common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -155,11 +156,22 @@ public Random current() {
             }
         }
         DIRECT_MEMORY_LIMIT = maxDirectMemory;
-        logger.debug("io.netty.maxDirectMemory: {} bytes", maxDirectMemory);
+        logger.debug("-Dio.netty.maxDirectMemory: {} bytes", maxDirectMemory);
+
+        int tryAllocateUninitializedArray =
+                SystemPropertyUtil.getInt("io.netty.tryAllocateUninitializedArray", 1024);
+        TRY_ALLOCATE_UNINITIALIZED_ARRAY = javaVersion() >= 9 && PlatformDependent0.hasAllocateArrayMethod() ?
+                tryAllocateUninitializedArray : -1;
+        logger.debug("-Dio.netty.tryAllocateUninitializedArray: {}", tryAllocateUninitializedArray);
```

Comment:
Do you want to log `tryAllocateUninitializedArray` or `TRY_ALLOCATE_UNINITIALIZED_ARRAY` here? Perhaps the latter would be more helpful for debugging? Or just log both?